doc/INSTALL.md

```diff
@@ -209,7 +205,7 @@ pkg_add git python gmake py3-pip libtool gmp
 pkg_add automake # (select highest version, automake1.16.2 at time of writing)
 pkg_add autoconf # (select highest version, autoconf-2.69p2 at time of writing)
 ```
-Install `mako` and `mrkd` otherwise we run into build errors:
+Install `mako` otherwise we run into build errors:
```

Comment:
for what we need `mako` in this case?

Reply:
`mako` is a templating language which is used to generate wire files; it's not related to `mrkd`, which is used to convert manpages from markdown into the roff file format which `man` requires.